docs/standard_library/modules.md

```diff
@@ -0,0 +1,30 @@
+# Modules
+
+The Deno Standard Library contains numerous modules which expose features and
+functionality to make a developer's work easier.
+
+Please note the Standard Library is currently unstable and some documentation is
+yet to be completed.
+
+- [Archive](/std/archive)
+- Async
+- [Bytes](/std/bytes)
+- [Date Time](/std/datetime)
+- [Encoding](/std/encoding)
+- [Flags](/std/flags)
+- [Format](/std/fmt)
+- [File System](/std/fs)
+- [Hash](/std/hash)
+- [HTTP](/std/http)
+- [Input Output](/std/io)
+- [Log](/std/log)
+- Mime
+- [Node](/std/node)
+- [Path](/std/path)
+- Permissions
+- Signal
+- [Testing](/std/testing)
+- Text Proto
+- [UUID](/std/uuid)
+- [WebAssembly System Interface](/std/wasi)
+- [Web Socket](/std/ws)
```

Comment:
There used to be such a list in `std/README.md`, it was removed and replaced with this section: https://github.com/denoland/deno/tree/master/std#documentation because it kept getting out of date and is redundant. Instead of having this page, maybe you could similarly just link to https://deno.land/std/?

Reply:
I'm happy to leave this if you really don't think it is needed, but do you not think this will aid with discoverability? Even if it creates a maintenance overhead.

I think there needs to be something in the manual to aid developers with learning Deno's standard library, and I'd see this as a start point rather than an endpoint.

Reply:
A link to https://deno.land/std/ where you linked this page is just as discoverable and contains even more information. Again, we had a redundant list like this and the maintenance overhead just wasn't being met so it was removed.

Reply:
Ok cool. 👍